lib/xcodeproj/scheme/environment_variables.rb

```diff
@@ -0,0 +1,117 @@
+require 'xcodeproj/scheme/xml_element_wrapper'
+
+module Xcodeproj
+  class XCScheme
+    VARIABLES_NODE = 'EnvironmentVariables'
+    VARIABLE_NODE = 'EnvironmentVariable'
+
+    # This class wraps the EnvironmentVariables node of a .xcscheme XML file. This
+    # is just a container of EnvironmentVariable objects. It can either appear on a
+    # LaunchAction or TestAction scheme group.
+    #
+    class EnvironmentVariables < XMLElementWrapper
+      # @param [REXML::Element,Array{EnvironmentVariable},Array{Hash{String => String}}] node_or_target
+      #   The 'EnvironmentVariables' XML node that this object represents.
```

Comment:
I could change the Hash keys to be strings instead of symbols as the Hash keys if that is more standard across the project, doesn't make a difference to me, seems how the enforced style is `=>` and not `:` for Hashes.

Alternatives are:
 - I could also just not allow initialization via the Hash, but it seems like a nice convenience. 
 - I could remove the ability to set enabled state through it. 

I didn't particularly like either of those options however.

I agree that it is a bit confusing having something that is already almost like a Hash to be reflected not as a hash, because of the desire to reflect the enabled state. For now, I'll document this expectation here unless you think something else is more appropriate.